test/integration/features/basic.feature

```diff
@@ -192,6 +192,24 @@ Feature: Basic
      When executing "minishift stop" succeeds
      Then Minishift should have state "Stopped"
 
+  Scenario: Stopping non existing VM
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift stop"
+     Then Minishift should have state "Does Not Exist"
+      And stdout should contain
+      """
+      The execution of this command requires an existing 'minishift' VM. However, there is currently none defined.
+      """
+
+  Scenario: Stopping stopped VM
+    Given Minishift has state "Stopped"
+     When executing "minishift stop"
```

Comment:
When executing "minishift stop" succeeds

this way we check the error code as well